syspurpose/src/syspurpose/sync.py

```diff
@@ -106,4 +110,4 @@ def send_syspurpose_to_candlepin(self, syspurpose_store):
                 print('Unable to update consumer with system purpose: %s' % err)
                 return False
 
-            return True
+            return True
```

Comment:
Bring that new line back if you're willing to do another push for whitespace.

Reply:
Done!